# Bulgarian translation for apport
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-02-23 23:16+0100\n"
"PO-Revision-Date: 2014-02-18 15:58+0000\n"
"Last-Translator: Atanas Kovachki <Unknown>\n"
"Language-Team: Bulgarian <bg@li.org>\n"
"Language: bg\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Събери информация за системата"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"За да изпратите съобщение за проблем и събиране на информация за системата, "
"е необходимо да се идентифицирате"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Доклади за системни грешки"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"Моля, въведете паролата си за достъп до докладите за проблеми на системни "
"програми"

#: ../apport/ui.py:167
msgid "This package does not seem to be installed correctly"
msgstr "Този пакет не изглежда да е инсталиран правилно"

#: ../apport/ui.py:177
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:207
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"В системата ви има инсталирани остарели версии на някои пакети. Моля, "
"надградете тези пакети и проверете, останал ли е проблема:\n"
"\n"
"%s"

#: ../apport/ui.py:370
msgid "unknown program"
msgstr "неизвестна програма"

#: ../apport/ui.py:373
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Извинете, програмата «%s» аварийно се затвори"

#: ../apport/ui.py:376 ../apport/ui.py:1924
#, python-format
msgid "Problem in %s"
msgstr "Проблем в %s"

#: ../apport/ui.py:381
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Вашият компютър няма достатъчно свободна памет, за да се анализира "
"автоматично проблема и да изпрати доклада на разработчиците."

#: ../apport/ui.py:441 ../apport/ui.py:459 ../apport/ui.py:632
#: ../apport/ui.py:641 ../apport/ui.py:878 ../apport/ui.py:1688
#: ../apport/ui.py:1885 ../apport/ui.py:1890
msgid "Invalid problem report"
msgstr "Невалиден доклад с проблем"

#: ../apport/ui.py:442
msgid "You are not allowed to access this problem report."
msgstr "Нямате достъп до този доклад за проблем."

#: ../apport/ui.py:450
msgid "Error"
msgstr "Грешка"

#: ../apport/ui.py:452
msgid "There is not enough disk space available to process this report."
msgstr "Няма достатъчно свободно място за обработка на този доклад."

#: ../apport/ui.py:490
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:492
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:503 ../apport/ui.py:608
msgid "Invalid PID"
msgstr "Невалиден PID"

#: ../apport/ui.py:504
msgid "The specified process ID does not exist."
msgstr ""

#: ../apport/ui.py:509
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:510
msgid "The specified process ID does not belong to you."
msgstr ""

#: ../apport/ui.py:565
msgid "No package specified"
msgstr "Не е указан пакет"

#: ../apport/ui.py:567
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr "Трябва да укажете пакет или PID. Вижте --help за повече информация."

#: ../apport/ui.py:595
msgid "Permission denied"
msgstr "Достъпа е отказан"

#: ../apport/ui.py:597
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Указаният процес не ви принадлежи. Моля, стартирайте тази програма от името "
"на собственика на процеса или като root потребител."

#: ../apport/ui.py:610
msgid "The specified process ID does not belong to a program."
msgstr "Указания идентификатор не принадлежи на програма."

#: ../apport/ui.py:634
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Симптомният скрипт %s не може да определи засегнат пакет"

#: ../apport/ui.py:642
#, python-format
msgid "Package %s does not exist"
msgstr "Пакета %s не съществува"

#: ../apport/ui.py:671 ../apport/ui.py:883 ../apport/ui.py:916
#: ../apport/ui.py:926
msgid "Cannot create report"
msgstr "Не може да бъде създаден доклад"

#: ../apport/ui.py:686 ../apport/ui.py:745 ../apport/ui.py:763
msgid "Updating problem report"
msgstr "Актуализиране на доклада за проблема"

#: ../apport/ui.py:688
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Вие не се явявате изпращач или абонат на доклада за проблем, възможно е "
"доклада да е дублиран или вече е затворен.\n"
"Моля създайте нов доклад като използвате «apport-bug»."

#: ../apport/ui.py:700
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Вие не сте създателят на този доклад за проблем. По-лесно е да маркирате "
"тази грешка като дублиран на друг, от колкото да преместите вашите коментари "
"и прикачени файлове в нова грешка.\n"
"\n"
"Освен това, ние ви препоръчваме да регистрирате нов доклад за грешка като "
"използвате «apport-bug» и създадете коментар в тази грешка в същият файл.\n"
"\n"
"Наистина ли искате да продължите?"

#: ../apport/ui.py:746 ../apport/ui.py:764
msgid "No additional information collected."
msgstr "Не е събрана допълнителна информация."

#: ../apport/ui.py:827
msgid "What kind of problem do you want to report?"
msgstr "Какъв вид проблем искате да докладвате?"

#: ../apport/ui.py:848
msgid "Unknown symptom"
msgstr "Неизвестен симптом"

#: ../apport/ui.py:849
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Симптомът «%s» е неизвестен."

#: ../apport/ui.py:885
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:901
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"След като затворите това съобщение, моля кликнете на прозореца на "
"приложението, за да докладвате за този проблем."

#: ../apport/ui.py:918 ../apport/ui.py:927
msgid "xprop failed to determine process ID of the window"
msgstr "xprop неуспя да определи идентификатора на процеса за прозореца"

#: ../apport/ui.py:942
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:943
msgid "Specify package name."
msgstr "Посочете име на пакета."

#: ../apport/ui.py:950 ../apport/ui.py:1083
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""
"Добавете допълнителен етикет към доклада си. Може да бъде указан няколко "
"пъти."

#: ../apport/ui.py:988
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:999
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Стартиране в режим на регистриране на грешка. Изисква --package и по избор --"
"pid, или само --pid. Ако не е подаден аргумент се показва списък с известни "
"симптоми. (Стандартно ако е подаден единствен аргумент.)"

#: ../apport/ui.py:1008
msgid "Click a window as a target for filing a problem report."
msgstr "Кликнете на целевият прозорец, за да попълните доклад за грешка."

#: ../apport/ui.py:1017
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Стартиране в режим на обновяване на грешка. Приема се аргумент по избор --"
"package."

#: ../apport/ui.py:1026
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Регистриране на доклад за грешка относно симптом. (Стандартно ако името на "
"симптома е подадено като единствен аргумент.)"

#: ../apport/ui.py:1035
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Посочете името на пакета в --file-bug режима. Това е по избор ако --pid e "
"посочен. (Стандартно ако името на пакета е подадено като единствен аргумент.)"

#: ../apport/ui.py:1046
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Определете работеща програма -- в -file-bug режим. Ако това е посочено, "
"доклада за грешка ще съдържа повече информация. (Стандартно ако pid е даден "
"като единствен аргумент.)"

#: ../apport/ui.py:1054
msgid "The provided pid is a hanging application."
msgstr "Посоченият идентификатор принадлежи на процес, който не отговаря."

#: ../apport/ui.py:1062
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Докладване на срив от даден .apport или .crash файл вместо от висящите в %s. "
"(Стандартно ако файла е подаден като единствен аргумент.)"

#: ../apport/ui.py:1072
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"В режим за докладване на грешки, запазете събраната информация във файл, "
"вместо да я докладвате. Този файл може да бъде докладван по-късно от друг "
"компютър."

#: ../apport/ui.py:1091
msgid "Print the Apport version number."
msgstr "Отпечатай номера на версията на Apport."

#: ../apport/ui.py:1253
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Ще се стартира «apport-retrace» в терминален прозорец и ще извърши проверка "
"на аварийното завършване на работата."

#: ../apport/ui.py:1257
msgid "Run gdb session"
msgstr "Стартирай gdb сесия"

#: ../apport/ui.py:1258
msgid "Run gdb session without downloading debug symbols"
msgstr ""
"Стартирай gdb сесия без изтегляне на символите за отстраняване на грешки"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1260
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr ""
"Актуализирай %s с пълно проследяване на символичното трасиране на стека"

#: ../apport/ui.py:1322
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1326
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1408 ../apport/ui.py:1421
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Проблемът се отнася за програма, която вече не е инсталирана."

#: ../apport/ui.py:1445
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Проблемът възникна с програмата %s, в която са внесени промени след нейното "
"аварийно завършване на работа."

#: ../apport/ui.py:1506 ../apport/ui.py:1629 ../apport/ui.py:1894
msgid "This problem report is damaged and cannot be processed."
msgstr "Този доклад за проблем е повреден и не може да бъде обработен."

#: ../apport/ui.py:1515
msgid "This report is about a package that is not installed."
msgstr ""

#: ../apport/ui.py:1523
msgid "An error occurred while attempting to process this problem report:"
msgstr "Възникна грешка при обработката на този доклад за проблем:"

#: ../apport/ui.py:1540
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1545
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1546
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1586
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1591
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1690
msgid "Could not determine the package or source package name."
msgstr ""
"Не може да бъде определено името на пакета или на пакета с изходния код."

#: ../apport/ui.py:1715
msgid "Unable to start web browser"
msgstr "Не може да се стартира уеб браузъра"

#: ../apport/ui.py:1716
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Не може да се стартира уеб браузъра, за да се отвори %s."

#: ../apport/ui.py:1840
msgid "Network problem"
msgstr "Мрежов проблем"

#: ../apport/ui.py:1844
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Не може да се свърже с базата данни за сривове, моля проверете връзката с "
"интернет."

#: ../apport/ui.py:1876
msgid "Memory exhaustion"
msgstr "Изчерпване на паметта"

#: ../apport/ui.py:1878
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Вашата система не разполага с достатъчно памет, за да обработи този доклад "
"за срив."

#: ../apport/ui.py:1929
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Проблемът не може да бъде докладван:\n"
"\n"
"%s"

#: ../apport/ui.py:1985 ../apport/ui.py:1997
msgid "Problem already known"
msgstr "Проблемът вече е известен"

#: ../apport/ui.py:1987
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"За този проблем е вече известно. Съществуващият доклад за него е отворен в "
"браузъра ви. Моля, проверете дали можете да добавите друга полезна "
"информация, която може да помогне на разработчиците."

#: ../apport/ui.py:1999
msgid "This problem was already reported to developers. Thank you!"
msgstr "Този проблем вече е бил докладван на разработчиците. Благодарим ви!"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Натиснете някой клавиш, за да продължите..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Какво бихте искали да направите? Вашите опции са:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Моля, изберете (%s)"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i байта)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:405
msgid "(binary data)"
msgstr "(двоични данни)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:198
msgid "Send problem report to the developers?"
msgstr "Да се изпраща ли доклад за грешка до разработчиците?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"След като доклада за проблема бъде изпратен, моля попълнете формата\n"
"в браузъра, който ще се отвори след това."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Изпрати доклад (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "&Локална проверка"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "&Прегледай доклада"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Съхрани файла на доклада за изпращане по-късно или копиране на друго място"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Отмени и &игнорирай бъдещи сривове на тази версия на програмата"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:326 ../bin/apport-cli.py:360
#: ../bin/apport-cli.py:383
msgid "&Cancel"
msgstr "&Отмени"

#: ../bin/apport-cli.py:267
msgid "Problem report file:"
msgstr "Отчетен файл на проблем:"

#: ../bin/apport-cli.py:273 ../bin/apport-cli.py:278
msgid "&Confirm"
msgstr "&Потвърди"

#: ../bin/apport-cli.py:277
#, python-format
msgid "Error: %s"
msgstr "Грешка %s"

#: ../bin/apport-cli.py:283 ../kde/apport-kde.py:455
msgid "Collecting problem information"
msgstr "Събиране на информация за проблема"

#: ../bin/apport-cli.py:285
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Събраната информация може да се изпрати на разработчиците, за да\n"
"подобрят приложението. Това може да отнеме няколко минути."

#: ../bin/apport-cli.py:301 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:504
msgid "Uploading problem information"
msgstr "Прехвърляне на информация за проблема"

#: ../bin/apport-cli.py:303
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Събраната информация се изпраща на системата за следене за грешки.\n"
"Това може да отнеме няколко минути."

#: ../bin/apport-cli.py:359
msgid "&Done"
msgstr "&Готово"

#: ../bin/apport-cli.py:365
msgid "none"
msgstr "нищо"

#: ../bin/apport-cli.py:367
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Избран: %s. Множество избори:"

#: ../bin/apport-cli.py:384
msgid "Choices:"
msgstr "Избори:"

#: ../bin/apport-cli.py:398
msgid "Path to file (Enter to cancel):"
msgstr "Път до файл (Enter за отказ):"

#: ../bin/apport-cli.py:404
msgid "File does not exist."
msgstr "Файлът не съществува."

#: ../bin/apport-cli.py:406
msgid "This is a directory."
msgstr "Това е папка."

#: ../bin/apport-cli.py:412
msgid "To continue, you must visit the following URL:"
msgstr "За да продължите, трябва да посетите следния адрес:"

#: ../bin/apport-cli.py:415
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Можете да стартирате браузъра сега или да копирате връзката в браузър на "
"друг компютър."

#: ../bin/apport-cli.py:421
msgid "Launch a browser now"
msgstr "Стартирай сега браузъра"

#: ../bin/apport-cli.py:437
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Няма предстоящи доклади за сривове. Пробвайте --help за повече информация."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Не поставяйте повече трасирания в доклада, а ги запишете в stdout."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Стартирай интерактивна gbg сесия с дъмпа от ядрото на доклада (-о "
"игнорирано; не презаписва доклада)."

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Съхрани модифициран доклад с дадено име на файл вместо променяне на "
"оригиналния доклад."

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Изтрий дъмпа от ядрото от доклада след регенериране трасирането на стека."

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Замени CoreFile в доклада"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Замени ExecutablePath в доклада"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Замени ProcMaps в доклада"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Изгради наново пакетната информация в доклада"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Създай временна среда, след това изтегли/инсталирай необходимите пакети и "
"символи за отстраняване на грешки; без тази опция се предполага, че "
"необходимите пакети и символи за отстраняване на грешки вече са инсталирани. "
"Аргумента указва базовата директория за конфигурация на пакетната система; "
"ако сте задали «system», ще се използват системните конфигурационни файлове, "
"но само в случай на извънредни ситуации, възникнали в изданието на "
"работещата в момента система."

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Докладвай за напредъка при сваляне/инсталиране на пакети във времената среда"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Добави временни отметки до съобщенията в дневника за групови операции"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""
"Кеш директория за съхраняване на пакетите, изтеглени във времената среда"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Директория за разопаковане на пакети. Когато повторение всички вече "
"изтеглени пакети ще се разопаковат в тази времена среда."

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Инсталирай във времената среда допълнителен пакет (може да се указва няколко "
"пъти)"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Път към файла с информация за автентикация на базата данни със сривовете. "
"Това се използва само когато се специфицира ID на срив, за да се качи "
"ретрасирано трасиране на стека (само ако нито един от -g, -o, nor -s са "
"специфицирани)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Показване на ретрасираните трасирания на стека и искане на потвърждение "
"преди изпращането им в базата данни със сривовете."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Път към дублираната sqlite база данни (стандартно: без проверка за дублиране)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr ""

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "Не може да се използва без -C без -S. Спиране."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "Да се изпрати ли прикаченото? [y/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Крайната директория съществува и не е празна."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "Вижте повече подробности в man страницата."

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "укажете името на лог файла за valgrind"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"използвай по-рано създадената директория за времената среда (SDIR) или я "
"създайте, ако тя не съществува"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"не създавай и не използвай директорията на времената среда, за допълнителни "
"символи за отстраняване на грешки, а разчитай само на установените символи "
"за отстраняване на грешки."

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"използвай по-рано създадената кеш директория (CDIR) или я създаде, ако тя не "
"съществува"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"докладвай прогреса за сваляне/инсталиране по врене на инсталирането на "
"пакетите в времената среда"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Грешка: %s не е изпълним файл. Спиране."

#: ../data/apportcheckresume.py:67
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:72
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:80
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Продължаването заби към края, но изглежда процедурата беше завършена "
"нормално."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Сега вашата система може да стане нестабилна и да се наложи да я "
"рестартирате."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Докладвай за проблем..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Докладвай за неизправност на разработчиците"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Извинете, приложението %s внезапно спря."

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Извинете, %s беше затворено неочаквано."

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:213 ../kde/apport-kde.py:265
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Извинете, възникна вътрешна грешка в %s."

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:206
msgid "Send"
msgstr "Изпрати"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:660 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:329
msgid "Show Details"
msgstr "Покажи детайлите"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:257 ../kde/apport-kde.py:275
msgid "Continue"
msgstr "Продължи"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "Приложението «%s» престана да отговаря."

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Програмата «%s» престана да отговаря."

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:221
#, python-format
msgid "Package: %s"
msgstr "Пакет: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:228
msgid "Sorry, a problem occurred while installing software."
msgstr "Извинете, възникна проблем при инсталирането на софтуер."

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:237
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "В приложението %s възникна вътрешна грешка."

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:244
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Приложението %s внезапно се затвори."

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:270
msgid "If you notice further problems, try restarting the computer."
msgstr ""
"Ако се сблъскате с този проблем в бъдеще, опитайте да рестартирате компютъра."

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:277
msgid "Ignore future problems of this type"
msgstr "Игнорирай бъдещи проблеми от този тип"

#: ../gtk/apport-gtk.py:664 ../kde/apport-kde.py:326
msgid "Hide Details"
msgstr "Скрий детайлите"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:545 ../kde/apport-kde.py:588
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr ""

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr ""

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Доклад за грешка"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Извинете, възникна вътрешна грешка.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr ""

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Игнорирай бъдещи проблеми с тази версия на програмата"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr ""

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Локална проверка"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr ""

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Събиране на информация за проблем</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Събира се информация, която може да помогне на разработчиците да поправят "
"проблема, който докладвате."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Качване на информация за проблем</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:506
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Събраната информация се изпраща към системата за следене на бъгове. Това "
"може да отнеме няколко минути."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport файл с доклад за срив"

#: ../kde/apport-kde.py:260
msgid "Leave Closed"
msgstr "Остави затворено"

#: ../kde/apport-kde.py:261 ../kde/apport-kde.py:425
msgid "Relaunch"
msgstr "Стартирай отново"

#: ../kde/apport-kde.py:354
msgid "Username:"
msgstr "Потребителско име:"

#: ../kde/apport-kde.py:355
msgid "Password:"
msgstr "Парола:"

#: ../kde/apport-kde.py:454
msgid "Collecting Problem Information"
msgstr "Събиране на информация за проблема"

#: ../kde/apport-kde.py:457
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Събраната информация може да се изпрати на разработчиците, за да подобрят "
"програмата. Това може да отнеме няколко минути."

#: ../kde/apport-kde.py:503
msgid "Uploading Problem Information"
msgstr "Прехвърляне на информация за проблема"
